css/main.scss

```diff
@@ -2,7 +2,21 @@ header#header {
 	display: none !important;
 }
 
-.oc-dialog {
+.modal-container__close {
+	display: none !important;
+}
+
+.file-picker__main {
+	.breadcrumb__actions {
+		display: none !important;
+	}
+}
+
+.dialog__actions {
+	justify-content: center !important;
+}
+
+/* .oc-dialog {
```

Comment:
Just remove the commented CSS, if seems to work without now so it is fine to just remove it.